support/cas-server-support-spnego/src/main/java/org/apereo/cas/config/SpnegoConfiguration.java

```diff
@@ -44,12 +46,9 @@
 @AutoConfiguration
 public class SpnegoConfiguration {
 
-    @RefreshScope(proxyMode = ScopedProxyMode.DEFAULT)
-    @Bean
-    @ConditionalOnMissingBean(name = "spnegoAuthentications")
-    public BeanContainer<Authentication> spnegoAuthentications(
-        final CasConfigurationProperties casProperties,
-        final ConfigurableApplicationContext applicationContext) {
+    private List<Authentication> buildSpnegoAuthentications(
```

Comment:
Returning a list generally is problematic for refreshing purposes. You should continue to return a BeanContainer, and then ask the container to give it back to you as a List

Reply:
No problem. Will do.